deps/wrapper.cc

```diff
@@ -1,12 +1,5 @@
 #include "wrapper.h"
 
-namespace julia {
-
-using namespace ray;
-using ray::core::CoreWorkerProcess;
-using ray::core::CoreWorkerOptions;
-using ray::core::WorkerType;
-
```

Comment:
these were moved to the header file right?  except for teh namespace.  why remove the namespace?